src/main/resources/templates/process-conf.xml

```diff
@@ -3,7 +3,7 @@
     <!-- For the absolute paths below, replace with a better mechanism, eg. config.properties param for base location -->
     <bean id="maximumBatchRowsDbProcess"
           class="com.salesforce.dataloader.process.ProcessRunner"
-          singleton="false">
+          scope="prototype">
```

Comment:
Could you explain about the changes?

Reply:
Spring framework upgrade necessitated this change. singleton="false" is not supported in the upgraded version of Spring. Replacement attribute is scope with value "prototype" per https://stackoverflow.com/questions/49879668/singleton-false-is-no-longer-supported-in-spring-4-3-8-need-solution